egs/wenetspeech/ASR/pruned_transducer_stateless2/asr_datamodule.py

```diff
@@ -410,13 +417,13 @@ def train_cuts(self) -> CutSet:
             logging.info("use lazy cuts")
             cuts_train = CutSet.from_jsonl_lazy(
                 self.args.manifest_dir
-                / "cuts_L_50_pieces.jsonl.gz"
+                / "cuts_L.jsonl.gz"
                 # use cuts_L_50_pieces.jsonl.gz for original experiments
             )
         else:
             cuts_train = CutSet.from_file(
                 self.args.manifest_dir
-                / "cuts_L_50_pieces.jsonl.gz"
+                / "cuts_L.jsonl.gz"
                 # use cuts_L_50_pieces.jsonl.gz for original experiments
```

Comment:
What are "original experiment" in the comment?

Reply:
It means that the `cuts_L_50_pieces.jsonl.gz` includes 1400+ hours. It refers the results showing above.